sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference}
 import org.apache.spark.sql.types._
 
 
+
```

Comment:
ParseException is a subclass of AnalysisException, which I think it is fine with my new class hierarchy. Do you want me still work on this for changing the coding style?